spec/ParseLiveQueryServer.spec.js

```diff
@@ -1128,15 +1220,28 @@ describe('ParseLiveQueryServer', function() {
     const requestId = 0;
 
     spyOn(Parse, 'Query').and.callFake(function() {
+      let shouldReturn = false;
       return {
         equalTo() {
+          shouldReturn = true;
           // Nothing to do here
+          return this;
+        },
+        containedIn() {
+          shouldReturn = false;
+          return this;
         },
         find() {
+          if (!shouldReturn) {
+            return Promise.resolve([]);
+          }
           //Return a role with the name "liveQueryRead" as that is what was set on the ACL
-          const liveQueryRole = new Parse.Role();
-          liveQueryRole.set('name', 'liveQueryRead');
-          return [liveQueryRole];
+          const liveQueryRole = new Parse.Role(
+            'liveQueryRead',
```

Comment:
wait, i'm confused, why wouldn't this be 'otherLiveQueryRead'?  i'm not sure what's going on here, or why you needed to change it.